SolomiiaLomnytska22/solomiia-training
# Added icons and created tooltip for "Remove" and "Edit" icons in the table.

**Refs:**
#14 (Add icons)

**Proposed changes:**
- Added icon to the "Add" button.
- Changed "Remove" and "Edit" buttons from the table to icons.
- Added tooltip component to the icons.

User interface before changes: 
<img width="1790" alt="Screenshot 2023-12-08 at 14 30 28" src="https://github.com/SolomiiaLomnytska22/solomiia-training/assets/136481621/846a63f2-904b-406e-88c1-4949f7aa2d06">

User interface after changes:
<img width="1790" alt="Screenshot 2023-12-08 at 14 31 47" src="https://github.com/SolomiiaLomnytska22/solomiia-training/assets/136481621/c28b57c7-7720-462d-8dcc-80bb53eb6615">

**Verification:**
To see new icons:
- Open the project.
- Notice that there is an icon in "Add" button and icons in the table instead of "Remove" and "Edit" buttons.

To see tooltips:
- Open the project.
- Hover your mouse over one of the icons in table.
- Notice tooltip appear near the icon.

All changes tested locally. New and already existing functionality is supported.

https://github.com/SolomiiaLomnytska22/solomiia-training/assets/136481621/e0a0abf9-ff65-4de5-a450-06befedd502a
